docs/explanations/purging_services.md

```diff
@@ -0,0 +1,79 @@
+# Disabling and purging services
+
+All services enabled using the Pro Client can be disabled using the command line:
+
+```bash
+$ sudo pro disable <service-name>
```

Comment:
Can we safely remove the $ symbols, or do they need to be here for clarity?

Reply:
Symbols are there for clarity.
`$` means you are running as a non-root user, and we want to show it and the `sudo` command there instead of `#` or nothing, to make it explicit that this is a command which requires privileges